azuki-core/src/main/kotlin/com/anaplan/engineering/azuki/core/runner/JUnitScenarioRunner.kt

```diff
@@ -195,15 +199,18 @@ class JUnitScenarioRunner<
             object : ReflectiveCallable() {
                 override fun runReflectiveCall(): Any {
                     try {
-                        val scenario = if (parameters == null) {
+                        val nonSpecialParameters = parameters
+                            ?.filterNot { it is Since || it is KnownBug || it is ToBeDone || it is Unsupported || it is RestrictTo }
```

Comment:
:pick: might have been easier to do 
`
it::class.qualifiedName?.startsWith("com.anaplan.engineering.azuki.core") == true
`